ember.md

```diff
@@ -183,3 +184,29 @@ CSS is permitted (and encouraged) in apps and addons under certain circumstances
 </div>
 
 ```
+
+<a name="updating-ember-apps"></a>
+## Updating Ember Apps
+
+1. Follow the steps under **Project Update** in the following guide: <https://github.com/ember-cli/ember-cli/releases>
+
+   1. Alternatively, you can use [ember-cli-update](https://github.com/kellyselden/ember-cli-update). However, make sure you are comfortable updating manually before using this method! 
+
+2. When you get to the last step, after running `ember init`, answer **yes** to all the prompts. 
+3. Make sure to delete `.jshintrc` files in both the /test and application root directories. Include `.eslintrc` files in the same directories instead. 
```

Comment:
This note is specific to updating to version 2.12.3
Updating to any version from 2.12.3 will not have to perform this step.

Reply:
@marialomidze how up-to-date is this change?